### ISSUE TITLE
Cannot install latest dev build (529) on Safari

### ISSUE THREAD

**Issue by LRFLEW** [YePpHa/YouTubeCenter#1948]
When trying to install the .safariextz file, Safari says "An error occurred while installing the extension “YouTube Center Developer Build”."

On the Console, the following messages appear:
`7/16/15 12:54:08.329 PM CoreServicesUIAgent[439]: Error -60005 creating authorization`
`7/16/15 12:54:08.399 PM sandboxd[430]: ([503]) com.apple.WebKit(503) deny file-read-data /Applications/Safari.app`
`7/16/15 12:54:08.402 PM sandboxd[430]: ([503]) com.apple.WebKit(503) deny file-read-data /Applications/Safari.app/Contents`

The latter two messages messages have a log file attached each. Since they're a little long to post here, I've posted them to pastebin [here](http://pastebin.com/D2wKmN88) and [here](http://pastebin.com/8BP5Y44S).

**Comment by miwu** [comment]
I'm also having this issue, though since I'm unfamiliar with Safari, I don't know how to provide logs :(

**Comment by DreamSeason** [comment]
Same error here. Hope this gets fixed asap.

**Comment by nihj09** [comment]
I have the same problem. I also don't know how to print out a log though.

**Comment by Ahmed707707** [comment]
I got error as well

**Comment by RSully** [comment]
Seeing this here too.

**Comment by digitalself** [comment]
Same issue here, with this in console:

17/07/2015 18:43:18.551 CoreServicesUIAgent[490]: Error -60005 creating authorization
17/07/2015 18:43:18.587 NML2NDeviceObserver[38004]: Executable to id(de.novamedia.Vodafone Mobile Broadband) keychain(None)
17/07/2015 18:43:18.590 NML2NDeviceObserver[38004]: Fatal error! Ivalid bundle. Application repository not available
17/07/2015 18:43:18.628 sandboxd[486]: ([32598]) com.apple.WebKit(32598) deny file-read-data /Applications/Safari.app
17/07/2015 18:43:18.636 sandboxd[486]: ([32598]) com.apple.WebKit(32598) deny file-read-data /Applications/Safari.app/Contents

**Comment by LRFLEW** [comment]
Still happens with Build 530

**Comment by Ahmed707707** [comment]
still the same :(

**Comment by digitalself** [comment]
Yup, issue persist in Build #530, same output in console

17/07/2015 23:46:54.325 NML2NDeviceObserver[16580]: Executable to id(de.novamedia.Vodafone Mobile Broadband) keychain(None)
17/07/2015 23:46:54.327 NML2NDeviceObserver[16580]: Fatal error! Ivalid bundle. Application repository not available
17/07/2015 23:46:54.329 com.apple.xpc.launchd[1]: (de.novamedia.VodafoneDeviceObserver) Service only ran for 0 seconds. Pushing respawn out by 10 seconds.
17/07/2015 23:46:55.353 CoreServicesUIAgent[487]: Error -60005 creating authorization
17/07/2015 23:46:55.000 kernel[0]: Sandbox: com.apple.WebKit(1113) deny file-read-data /Applications/Safari.app
17/07/2015 23:46:55.000 kernel[0]: Sandbox: com.apple.WebKit(1113) deny file-read-data /Applications/Safari.app/Contents

**Comment by dastefster** [comment]
I'm also still having this issue with #530 on Safari 8.0.7. Can't even download the extension by just clicking it, as I used to be able to, only way to DL is by alt clicking the link and saving the file, which doesn't install.

**Comment by nihj09** [comment]
I'm having the same issue with #530 now. Hopefully safari users won't be left behind...

**Comment by Shoemakerlevi** [comment]
me too

**Comment by ireun** [comment]
@YePpHa Can you look at this?

**Comment by YePpHa** [comment]
@ireun sure

Anyway, can any of you install the older versions of the developer version?

**Comment by digitalself** [comment]
Yep, I reinstalled build #528 without a hitch.

**Comment by YePpHa** [comment]
Seems like the issue is that I need to renew the certificate I'm using as it has expired. Another thing is that I now need to enroll in some kind of Apple Developer Program Enrollment. However, the problem is that I can't complete the sign up as there occurred some kind of unknown error and another little thing is that it now costs a little sum of money (not an issue). I will try to see if I can do it tomorrow or sometime later this next week.

**Comment by dastefster** [comment]
FYI, I also re-installed 528 and it worked fine!

**Comment by miwu** [comment]
@YePpHa Out of curiosity, is the safari extension just the userscript in an injection wrapper, or is it substantially different? I've temporary using the userscript with tampermonkey on Safari and it seems to be working without problem.

**Comment by YePpHa** [comment]
@miwu You're correct that the Safari extension is simply an injection wrapper. I made the extension available for the users convenience and not because of some limiting factor. However, the Firefox addon and Chrome extension actually uses some of its browser's extensui

**Comment by Shoemakerlevi** [comment]
still the same on 531 :(

**Comment by DreamSeason** [comment]
Any updates?

**Comment by hahnalexander** [comment]
I hate Apple for having developers sign up and pay. Actually funny that you don't mind to pay but are not able sign up. Crazy stuff. Any news?

**Comment by harveyadcock** [comment]
Still not working with version 531, any chance of someone uploading version 528 as I uninstalled it to clean install 531 to see if that was the issue before reading this thread :/

**Comment by ireun** [comment]
Sure, you can donwload it here: https://github.com/YePpHa/YouTubeCenter/blob/83300554b64355de1e3d410b3d53009335cccaf1/dist/YouTubeCenter.safariextz

**Comment by camilohoyos** [comment]
I followed the idea of @miwu and began to use [Tampermonkey] (https://tampermonkey.net) with the [last Userscript for Youtube Center] (https://github.com/YePpHa/YouTubeCenter/wiki/Developer-Version). It works flawlessly, and I stopped worrying about the Safari version not be updated. This script is great but Safari updates have being, and it seems they will continue to be cumbersome for @YePpHa.

**Comment by hahnalexander** [comment]
@camilohoyos thanks, Tampermonkey works perfect

**Comment by nihj09** [comment]
@camilohoyos Yes, I also downloaded Tampermonkey and downloaded the latest (531) userscript.js for youtube center.  Works great!
